docs/packs/packs-format.md

```diff
@@ -134,47 +138,53 @@ A supported partner pack metadata contents for example:
 }
 ```
 
-
 ### Content Packs Versioning
+
 Pack versions have the following format MAJOR.MINOR.REVISION:
+
    1. **Revision** when you make backwards compatible bug fixes.
    2. **Minor** when you add functionality in a backwards compatible manner.
    3. **Major** when you make incompatible API changes or revamping the pack by adding to it a lot of new backwards compatible functionality.
 
-
 ### README.md
+
 The file contains a general explanation for the pack and you are free to add any information relevant for the pack. For more details refer to the [Pack Documentation](../documentation/pack-docs) page.
 
 ### .secrets-ignore
+
 This file will be used while running the `demisto-sdk secrets`([explanation](https://github.com/demisto/demisto-sdk/blob/master/demisto_sdk/commands/secrets/README.md)), we will determine the file and will
  use it as an allow list of approved words for your PR.
 
 **Note**: We use `demisto-sdk secrets` as part of our pre-commit hook to check that possible secrets in the PR aren't exposed to a public repository.
 
 ### .pack-ignore
+
 1) This file allows ignoring linter errors while lint checking and ignoring tests in the test collection.
 
    To add ignored tests/linter errors in a file, first, add the file name to the **.pack-ignore** in this format
-```
-[file:integration-to-ignore.yml]
-```
 
-On the following line add `ignore=` flag, with one or more comma-separated values:
-* `auto-test` - ignore test file in the build test collection.
-* `linter code` e.g., IN126 - ignore linter error codes.
+    ```text
+    [file:integration-to-ignore.yml]
+    ```
+
+    On the following line add `ignore=` flag, with one or more comma-separated values:
+
+    * `auto-test` - ignore test file in the build test collection.
+    * `linter code` e.g., IN126 - ignore linter error codes.
 
 2) By default, unit-tests of scripts/integrations are running without a docker network.
 
    In case one of the integrations/scripts inside a pack needs a network during the unit-tests run, this can be done in this format
 
-```
-[tests_require_network]
-integration-id-1
-script-id-1
-```
+    ```text
```

Comment:
why the indentation?

Reply:
because it belongs to the section number 2